open-metadata-implementation/access-services/data-platform/data-platform-api/src/main/java/org/odpi/openmetadata/accessservices/dataplatform/responses/DataPlatformOMASAPIResponse.java

```diff
@@ -24,7 +24,8 @@
 @JsonSubTypes(
         {
                 @JsonSubTypes.Type(value = GUIDResponse.class, name = "GUIDResponse"),
-                @JsonSubTypes.Type(value = VoidResponse.class, name = "VoidResponse")
+                @JsonSubTypes.Type(value = VoidResponse.class, name = "VoidResponse"),
+                @JsonSubTypes.Type(value = RegistrationRequestBody.class, name = "SoftwareServerCapability"),
```

Comment:
You could separate Response class from Request class. We could use the same naming for the same concept, maybe we should discuss about using softwareServerCapability instead of registration naming for consistency across all OMASs.